package.json

```diff
@@ -22,7 +22,7 @@
     "lodash.remove": "4.7.0",
     "lodash.set": "4.3.2",
     "mkdirp": "0.5.1",
-    "selenium-webdriver": "https://github.com/sitespeedio/selenium/archive/v3.6.0.tar.gz",
+    "selenium-webdriver": "https://github.com/ncalexan/selenium/archive/v3.6.0-android.tar.gz",
```

Comment:
This seems fine for now, but I think we should probably have a mozilla fork of this.

Reply:
I agree.  I will fork to `mozilla` now and see how merging my changes into their clone works.

Reply:
This was trivial because `sitespeedio/selenium` has not advanced (and `sitespeedio/browsertime` still uses `v3.6.0`.)